lib/blemulator/internal/bridge/constants.dart

```diff
@@ -6,6 +6,33 @@ abstract class ChannelName {
   static String downstream = "$_base/toJava";
 }
 
-abstract class MethodName {
+abstract class DownstreamMethodName {
   static String simulate = "simulate";
-}
\ No newline at end of file
+}
+
+abstract class SimulationDownstreamMethodName {
+  static const String publishScanResult = "publishScanResult";
+}
+
+abstract class UpstreamMethodName {
```

Comment:
Just a suggestion. Maybe we should get rid of all downstream/upstream names since we partially did so?